lib/Installation/NtpConfiguration/NtpConfigurationPage.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The module provides interface to act on the
+#          ntp Configuration page.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Installation::NtpConfiguration::NtpConfigurationPage;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+
+sub init {
+    my $self = shift;
+    $self->SUPER::init();
+    $self->{txb_ntp_server} = $self->{app}->textbox({id => '"Y2Caasp::Widgets::NtpServer"'});
+    return $self;
+}
+
+sub is_shown {
+    my ($self) = @_;
+    return $self->{txb_ntp_server}->exist();
+}
+
+sub get_ntp_config {
+    my ($self, $ntpserver) = @_;
```

Comment:
You're not using `$ntpserver`.

Reply:
No, I don't use this. Deleted it. Thanks.